source/scss/_library/_objects.forms.scss

```diff
@@ -12,3 +12,114 @@
 .is-valid {
   border-color: $c-valid !important;
 }
+
+/**
+ * Search entry
+ */
+.c-search {
+  display: flex;
+  justify-content: space-between;
+
+  &__button {
+    height: 100%;
+    padding: 10px;
+  }
+}
+
+// Heading search menu
+.c-search--top {
+  height: 50px;
+  width: 265px;
```

Comment:
since this applied to the component itself, shouldn't the width be set to 100%, and the 265px get handled by a parent container?